src/FSharp.Formatting.Common/YaafFSharpScripting.fs

```diff
@@ -464,26 +463,29 @@ module internal ArgParser =
         else
             None
 
+    [<return: Struct>]
```

Comment:
@nojaf here is an example where  there is a Struct boolean. As boolean is a tiny field, there is no point of wrapping it to inside a separate class, like the original code did.